frameworks/Scala/fintrospect/benchmark_config.json

```diff
@@ -1,23 +1,28 @@
 {
   "framework": "fintrospect",
-  "tests": [{
-    "default": {
-      "orm": "Raw",
-      "database_os": "Linux",
-      "setup_file": "setup",
-      "json_url": "/json",
-      "plaintext_url": "/plaintext",
-      "port": 9000,
-      "approach": "Realistic",
-      "classification": "Micro",
-      "database": "None",
-      "framework": "fintrospect",
-      "language": "Scala",
-      "platform": "Netty",
-      "webserver": "None",
-      "os": "Linux",
-      "display_name": "fintrospect",
-      "notes": ""
+  "tests": [
+    {
+      "default": {
+        "orm": "Raw",
+        "database_os": "Linux",
+        "setup_file": "setup",
+        "json_url": "/json",
+        "fortune_url": "/fortunes",
+        "plaintext_url": "/plaintext",
+        "port": 9000,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "MySQL",
+        "db_url": "/db",
```

Comment:
Should this be here? It doesn't look like you have a DB test implemented. If that's the case, I'd take this line out.